### PR TITLE
GAA-1_fieldset_and_legend

### PR DESCRIPTION
**Pull request by bobmeredith**
To do (delete all that do not apply):

 - [Y] Changelog entry added.
 - [Y] (if adding HTML) Change is accessible to the standards we subscribe to.

### CODE REVIEW

**Review by depsiatwal — COMMENTED** [review]
Link to PR please in CL

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 35.20.0
```
Comment (lewis-coulson-dit) [inline CHANGELOG.md:3]: Think you need to put the PR URL here like the version below

File: directory_components/templates/directory_components/form_widgets/form_field.html [hunk 2]
```diff
@@ -1,10 +1,24 @@
...
-  {% if field.errors %}
-    <div class="error-message">{{ field.errors }}</div>
-  {% endif %}
-  {% if field.help_text %}
-    <span class="form-hint">{{ field.help_text }}</span>
-  {% endif %}
-  {{ field }}
+	{% if field.field.widget.is_grouped %}
+	<fieldset>
+		{% if field.label %}
```
Comment (lewis-coulson-dit) [inline directory_components/templates/directory_components/form_widgets/form_field.html:4]: looks like more horizontal spacing. Personally I don't mind this but just thinking maybe some linting may break as a result?

Comment (bobmeredith) [reply directory_components/templates/directory_components/form_widgets/form_field.html:4]: Yeah, I think my prettier is messing it up.
Will sort.

**Review by lewis-coulson-dit — APPROVED** [review]
Beautiful!

**Pull request merged**